sematic/db/migrations/20221208183307_create_server_settings_file.py

```diff
@@ -0,0 +1,21 @@
+# Standard Library
```

Comment:
Please update `changelog.md` and `developer-docs/README.md` to remove the previous manual copying instructions.